templates/email/default/_email_settings.html

```diff
@@ -25,6 +25,7 @@
 header_text_color = color_black
 header_padding = "15px 20px" # a full CSS padding property (eg: top/right/bottom/left)
 
+logo_file = "email-logo.gif"
```

Comment:
You can remove the `DEFAULT logo_file = 'email-logo.gif';` line from `_email_top.html` then.